tests/behaviour/concept/type/BUILD

```diff
@@ -25,6 +25,16 @@ rust_test(
     data = ["@vaticle_typedb_behaviour//concept/type:entitytype.feature"],
 )
 
+rust_test(
+    name = "relationtype",
```

Comment:
can we make these with underscore, or is there a reason not to?
I'm actually thinking the BDD files should also using undescore... `entity_type.feature`